Stop-words routes changes and updated some wording (#107)
meilisearch/documentation: references/stop_words.md

# Stop words

> The stop-word list is part of the [settings][1] category.

Reviewer: ```suggestion
The stop-words list is part of the [settings][1] category.
```

[1]: /references/settings.md
::: tip

> The stop-word list is considered as one resource and has the REST routes in line with this logic.

Reviewer: ```suggestion
The stop-words list is considered as one resource and has the REST routes in line with this logic.
```

```

## Create stop-words list

<RouteHighlighter method="POST" route="/indexes/:uid/settings/stop-words" />

Create the list of [stop-words](/advanced_guides/stop_words).

::: warning

> **If one already existed it will be overide.**

Reviewer: ```suggestion
**If one already exists, it will be overridden.**
```
